mobile/asa-go/src/components/HamburgerMenu.tsx

```diff
@@ -95,21 +77,74 @@ export const HamburgerMenu = ({
               <CloseIcon />
             </IconButton>
           </Grid>
-          <Grid>
-            <ActualForecastControl runType={runType} setRunType={setRunType} />
-          </Grid>
-          <Grid>
-            <WPSDatePicker date={date} updateDate={updateDate} />
-          </Grid>
-          <Grid>
-            <FireCenterDropdown
-              fireCenterOptions={fireCenterOptions}
-              selectedFireCenter={selectedFireCenter}
-              setSelectedFireCenter={setSelectedFireCenter}
-              setSelectedFireShape={setSelectedFireShape}
-              setZoomSource={setZoomSource}
-            />
-          </Grid>
+          <List
+            sx={{
+              width: "100%",
+              "& .MuiListItemButton-root": {
+                width: "100%",
+                justifyContent: "flex-end",
+              },
+            }}
+          >
+            <ListItemButton
```

Comment:
:nit - CodeCov would probably complain about duplicate code for all these `ListItemButton`s. If we need to do any additional styling on these list items it would be easier to have an array of objects that we `map` into `ListItemButton` components.

Reply:
Good call, addressed in: https://github.com/bcgov/wps/pull/4563/commits/4a9e634065634721c86f3cbf2f791e2e354582ce